### PR TITLE
Cat cake. A small update.

### PR DESCRIPTION
**Pull request by Nyranu**
## About the PR
This small PR will allow you to put a cakecat on your head, as well as add sprites holding it in your hand and on your head. I think it will add more cute mini elements, and will also be a great reward for cooking it.

## Why / Balance
I have sometimes seen questions of this type in looc: “Why can't you put it on your head like a moth” 

## Media
Screenshot_225
Screenshot_224

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: The ability to put a cakecat on your head.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 1]
```diff
@@ -578,6 +578,12 @@
     horizontalRotation: 0
   - type: Item
     size: Ginormous
+  - type: Clothing
+    quickEquip: false
+    sprite: Mobs/Pets/cakecat.rsi
+    equippedPrefix: 0
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:584]: for what prefix if it's only one cak in folder

File: Resources/Textures/Mobs/Pets/cakecat.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Update by Nyranu. The original PR from TheShuEd",
```
Comment (lzk228) [inline Resources/Textures/Mobs/Pets/cakecat.rsi/meta.json:4]: you should put here copyright from `Resources/Textures/Mobs/Pets/cat.rsi/meta.json` and new copyright for inhand and equipped states

**Review by lzk228 — CHANGES_REQUESTED** [review]
since cak moved to new .rsi, remove states from old folder

**Comment by lzk228** [comment]
read [pr guidelines](https://docs.spacestation14.io/en/getting-started/pr-guideline) and fix your cl

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 3]
```diff
@@ -578,10 +578,15 @@
     horizontalRotation: 0
   - type: Item
     size: Ginormous
+    inhandVisuals:
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:581]: can be changed to `heldPrefix: cak`

**Review by lzk228 — CHANGES_REQUESTED** [review]

**Review by lzk228 — APPROVED** [review]
not sure about copyright but seems good to me

**Comment by LankLTE** [comment]
> maybe make the cl more in-universe? maybe:
> 
> "Extended monitoring of feline pastroid specimens has led to the caks becoming more comfortable with researchers, allowing for crew to hold them and wear them on their heads."

Do not do this. The changelog should probably be removed outright as it's fairly minor (per the [pull request guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html#writing-an-effective-changelog))

**Comment by Emisse** [comment]
cl is fine rn

**Pull request merged**